### DISCUSSION TITLE
Axis Horn Speaker - Audio talkback via ONVIF or other standard protocols

### DISCUSSION THREAD

**Discussion by nost18** [AxisCommunications/discussions#188] [VAPIX] [answered]
We want to integrate the Axis Horn speaker into our application. For this we need the functionality Audio Talkback. However, according to the documentation, audio talkback is only possible via SIP protocol. Is this true or are there other ways, e.g. via ONVIF?

We want to avoid to implement SIP in our application.

**Comment by vivekatoffice** [comment, marked as answer]
Related links:
[288](https://github.com/orgs/AxisCommunications/discussions/288)
[axis-media-control-sdk](https://www.axis.com/developer-community/axis-media-control-sdk)
